dev-packages/scripts/chescript.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
```

Comment:
why is it written in js and not in typescript ?

Reply:
It's was just simple, but OK I rewrite it with TS